CharlieBackend.AdminPanel/Controllers/AccountController.cs

```diff
@@ -15,7 +15,6 @@
 
 namespace CharlieBackend.AdminPanel.Controllers
 {
-    [Route("admin/account")]
```

Comment:
leave at least [Route("[controller]")], because otherwise, Login method would be routed to /login